src/state/store.js

```diff
@@ -24,22 +24,28 @@ function getDefaultState(
   // Storybook's URL.)
   const firstPathComponent = window.location.pathname.split('/')[1];
 
+  const searchProducts = {
+    firefox: 'firefox',
+    fog: 'firefox',
+    fenix: 'fenix',
+  };
+
   if (Object.keys(productConfig).includes(firstPathComponent)) {
     state.product = firstPathComponent;
   } else {
-    state.product = undefined;
+    state.product = 'firefox';
   }
 
   state.probeName = '';
   state.ref = getFromQueryString('ref') || '';
   state.hov = getFromQueryString('hov') || '';
   state.timeHorizon = getFromQueryString('timeHorizon') || 'MONTH';
   state.route = {};
-  state.searchProduct = state.product || 'firefox';
+  state.searchProduct = searchProducts[state.product] || 'firefox';
```

Comment:
Is the fallback to `firefox` here redundant since we have `state.product` defaulting to `firefox` on L36 already?